patterns/organisms/forms/sample-form.php

```diff
@@ -0,0 +1,34 @@
+<?php
```

Comment:
is this custom form approach better than to use Gravity Forms?

Reply:
is not related with gravity forms, is just an example hot to integrate bootstrap forms in and organism